core/src/components/modal/modal.tsx

```diff
@@ -270,6 +277,12 @@ export class Modal implements ComponentInterface, OverlayInterface {
     }
   }
 
+  breakpointsChanged(breakpoints: number[] | undefined) {
+    if (typeof breakpoints !== 'undefined') {
```

Comment:
Any particular reason for this? Doing `breakpoints !== undefined` should be sufficient